articles/open-source-licencses/index.md

```diff
@@ -0,0 +1,108 @@
+# Introduction to open source licenses
+
+An essential aspect of a developer's job is assigning the appropriate license to the work done. The license ensures the work gets the due credit it deserves. They help in defining the boundaries of usage by an external party. The open-source community is responsible for the tremendous growth the world of technology has seen. Every aspect of technology, be it distributed computing, virtualization, machine learning, or web development, has been transformed through open source community contributions. 
+
+However, there are scenarios where a small organization or a group of developers would push code onto the web. The reason behind making the code public can be many. It might be to educate the public and help bridge the gap to learn the technology, or it might be a way to let others use the software commercially without any restrictions. This article will go over some of the most often used license agreements and understand their utilities.
+
+### Table of contents
+
+
+1. How to license
+2. MIT license
+3. GNU General Public License(GPL)
+4. Apache License
+5. Creative Commons Attribution
+6. The Unlicense
+7. Codebases with no license included
+
+### How to license
+
+After going through this article, you will know which license would be the best fit for your use case. Whichever license you choose, you must include the license along with the source code. There are a couple of ways to ensure the license is included.
+
+1. Go to the link of the licencse page and find the license. Include a link to the license and mention that the work presented is licensed under the respective license.
+2. The second way is to include the markdown version of the license in a file called LICENSE.md. GitHub recognizes this file and displays the type of license on the right panel of the repository page.
```

Comment:
>  LICENSE.md

This is being recognized as a link by markdown and is redirecting to a website named license.md when I click on it. Could you please make sure this doesn't happen, perhaps by adding backticks?

```md
`license.md`
```